swap_meet/vendor.py

```diff
@@ -0,0 +1,72 @@
+from swap_meet.item import Item
+from swap_meet.electronics import Electronics
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+
+class Vendor:
+    def __init__(self, inventory=None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+    
+    def add(self,item_name):
+        self.inventory.append(item_name)
+        return item_name
+
+    def remove(self,item_name):
+        if item_name in self.inventory:
+            self.inventory.remove(item_name)
+        else:
+            return False
+        return item_name
+
+    def get_by_category(self, category):
+        category_list = []
+        for items in self.inventory:
+            print(items.category)
```

Comment:
oop! don't forget to remove any print statements used for debugging before submitting your final version!